portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+/**
+ * Handles requests sent to the /hello URL. Try running a server and navigating
+ * to /hello!
+ */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
-}
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        ArrayList<String> arr = new ArrayList<String>(); // Creates an arrayList
```

Comment:
optional: In practice you'll have a lot more libraries at your disposal, like (my fave) [`ImmutableList`](https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html) that allow you to create things in-line. 

One way to do this with your current Java libraries is using `Arrays.asList()`, example:

```
ArrayList < String > facts = new ArrayList < String > (Arrays.asList(
            "Well, you have found my flabbergast button, and guess what? You pressed it. -Schmidt, New girl",
            "Another quote...",
            //...
        ));
```